cluster.go

```diff
@@ -605,6 +605,23 @@ func (c *Cluster) ID(ctx context.Context) *api.ID {
 	}
 }
 
+// GC performs garbage collection in the cluster
+func (c *Cluster) RepoGC(ctx context.Context) *api.IPFSRepoGc {
+	_, span := trace.StartSpan(ctx, "cluster/GC")
+	defer span.End()
+	ctx = trace.NewContext(c.ctx, span)
+
+	// ignore error since it is included in response object
+	ipfsGC, err := c.ipfs.RepoGC(ctx)
```

Comment:
Here, instead of calling the local IPFS RepoGC, you will need to make a broadcast request to every peer. There are examples in this file (anything with `rpc.MultiCall`). Otherwise leave a comment and I or someone else will fix it when the rest is ready.